pkg/kubectl/cmd/apply_set_last_applied.go

```diff
@@ -151,7 +151,7 @@ func (o *SetLastAppliedOptions) Validate() error {
 		if err != nil {
 			return err
 		}
-		patchBuf, diffBuf, patchType, err := editor.GetApplyPatch(info.Object, scheme.DefaultJSONEncoder())
+		patchBuf, diffBuf, patchType, err := editor.GetApplyPatch(info.Object.(runtime.Unstructured))
```

Comment:
this commit does not belong to this PR, does it?

Reply:
> this commit does not belong to this PR, does it?

Split out and merged here: https://github.com/kubernetes/kubernetes/pull/63398 .  Sorry about the noise.